.yamllint.yml

```diff
@@ -1,6 +1,9 @@
 ---
 extends: default
 
+ignore: |
+  doc/jupyterlite_environment.yml
+
```

Comment:
The file looks perfectly valid; I don't see any reason to ignore it.

Reply:
Yes, it was asking for some frontmatter, so I've addressed based on how other YAML files do it in 451ae62a05c2d183b565823e39b8d46f1b5b7a78.